### PR TITLE
buildFHSEnv: passthrough dconf database for programs

### LINKED ISSUE #353975: buildFHSEnv misses dconf profile
### Describe the bug
Program built with build-fhsenv-bubblewrap like Lutris misses profile created via `programs.dconf.profiles`

### Steps To Reproduce
1. Install Lutris
2. Create `programs.dconf.profiles` and apply settings like theme
3. Run Lutris

### Expected behavior
dconf profile linked in FHS environment of Lutris

### Additional context
build-fhsenv-bubblewrap should link /etc/dconf into FHS [like these](https://github.com/NixOS/nixpkgs/blob/nixos-24.05/pkgs/build-support/build-fhsenv-bubblewrap/default.nix#L55)

### Metadata

```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.32, NixOS, 24.05 (Uakari), 24.05.20240531.805a384`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/i1aw9jjgxcvyd642s12kw3iasmarwd42-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by ghost**
Closes #353975

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Would be good to also update the commit title.

File: pkgs/build-support/build-fhsenv-bubblewrap/default.nix [hunk 1]
```diff
@@ -96,6 +96,7 @@ let
       "ssl/certs"
       "ca-certificates"
       "pki"
+      "dconf"
```
Comment (Atemu) [inline pkgs/build-support/build-fhsenv-bubblewrap/default.nix:99]: Please rename the commit and also add a comment here that briefly explains why this is necessary. (i.e. "Required for `programs.dconf.profiles.user.databases` to function.").

**Comment by Atemu** [comment]
You must squash/fixup the commits and then force push to rename the commit.

**Comment by dukzcry2** [comment]
@Atemu I thought it's a part of merge :open_mouth:

**Comment by Atemu** [comment]
We don't usually squash merge. I don't know why anyone would ever want to do that either as it discards valuable historical information.

We instead clean up commit histories in PRs as they're being reviewed and then do a regular old merge commit.

Please see the contributor's guide on commit guidelines for more info.

**Pull request closed without merging**